lib/evm/eei.ts

```diff
@@ -554,7 +567,7 @@ export default class EEI {
   }
 }
 
-function trap (err: string) {
+function trap (err: ERROR) {
```

Comment:
Yeah, seems that these kind of typings are also very much a driver for consistency. Great to have this error cleanup.

Reply:
Or at least some start. 😋